src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs

```diff
@@ -252,15 +252,15 @@ private void UpdateTextBoxView()
 							_rectsChanged = true;
 							rectangle.Fill = SelectionHighlightColor;
 						}
-						if (rectangle.Width != Math.Ceiling(rect.Width))
+						if (rectangle.Width != rect.Width)
```

Comment:
Should we be comparing with epsilon? I'm not sure if precision matters here.

Reply:
I don't think so. rectangle.Width is wide enough to fit rect.Width without truncation.

Reply:
I'm mostly mentioning this if there are differences in size computation, particularly if we'd miss the exact same value.

Reply:
The calculations should be the exact same as long as the text hasn't changed since the offsets are just the addition of each glyph's `XAdvance` (not exactly but almost).